x-pack/plugins/security_solution/public/management/cypress/tasks/alert_details_actions.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
These are screen specific actions - I think they should be moved to the existing `screens/alerts.ts`.